util/jsonutil/jsonutil.go

```diff
@@ -0,0 +1,57 @@
+package jsonutil
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+)
+
+var comma = []byte(",")[0]
+
+func DropElement(extension []byte, elementName string) ([]byte, error) {
```

Comment:
Nice use of the JSON library to do the minimal decoding needed to snip out the JSON object.